bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +46,99 @@
 #}
 {% block dashboard_intro %}
 
-<p>
+<p class="intro">
 {% if LANG in ['en-US'] %}
-The Firefox new tab page now includes Tiles, providing users easy access to interesting, relevant content, whether it’s the sites they visit most or new sites for them to discover.
+  {{ _('The New Tab page offers convenience and more control of your browsing experience. Turn on Suggested Sites for a personalized experience of frequently used sites and relevant recommendations every time you open a New Tab.') }}
 {% else %}
-{{ _('With Enhanced Tiles and Directory Tiles, Firefox provides users easy access to interesting, relevant content, whether it’s the sites they visit most or new sites for them to discover.') }}
+  {{ _('With Enhanced Tiles and Directory Tiles, Firefox provides users easy access to interesting, relevant content, whether it’s the sites they visit most or new sites for them to discover.') }}
 {% endif %}
 </p>
 
-<h3>{{ _('What are Tiles?') }}</h3>
-
-<p>
+<section class="tls-personalize">
 {% if LANG in ['en-US'] %}
-When you open a new tab in Firefox, you see several rectangles that link to different websites.
-Others call them squares, windows or bookmarks; we call them Tiles.
+  <h2>{{ _('Personalize your New Tab experience') }}</h2>
+  <p>{{ _('You are in complete control of your New Tab page. Customize your experience by rearranging, pinning or removing sites, and set up New Tab just the way you want it.') }}</p>
+  <ol>
+    <li>{{ _('Drag a site to exactly where you want it.') }}
+      <img src="{{ static('img/firefox/tiles/drag-site.png') }}" alt="{{ _('Screenshot') }}">
+    </li>
+    <li>{{ _('Once you\'ve found the perfect spot for your site, pin it to make it stay.') }}</li>
+    <li>{{ _('Make a site disapear by deleting it from your New Tab page.') }}</li>
+  </ol>
+  <img src="{{ static('img/firefox/tiles/pin-site.png') }}" alt="{{ _('Screenshot') }}">
+  <p>
+    <a href="https://support.mozilla.org/en-US/kb/new-tab-page-show-hide-and-customize-top-sites">{{ _('Learn more about customizing New Tab') }}</a>
+  </p>
 {% else %}
-{{ _('When you open a new tab in Firefox, you see several rectangles that link to different websites.') }}
-{{ _('Some people call them squares, windows or bookmarks; we call them Tiles.') }}
+  <h2>{{ _('What are Tiles?') }}</h2>
+  <p>{{ _('When you open a new tab in Firefox, you see several rectangles that link to different websites.') }}</p>
+  <p>{{ _('Some people call them squares, windows or bookmarks; we call them Tiles.') }}</p>
```

Comment:
I do not see this content in the new copy document. Should this remain? Perhaps targeted at a different locale? Perhaps it is just the diff that makes this seem out of place but, just wanted to double check.